tests/integration/cli/action/test_action_classification.py

```diff
@@ -64,6 +64,20 @@ def test_otx_export(self, template, tmp_dir_path):
         tmp_dir_path = tmp_dir_path / "action_cls"
         otx_export_testing(template, tmp_dir_path)
 
+    @e2e_pytest_component
+    @pytest.mark.skipif(TT_STABILITY_TESTS, reason="This is TT_STABILITY_TESTS")
+    @pytest.mark.parametrize("template", templates, ids=templates_ids)
+    def test_otx_export_fp16(self, template, tmp_dir_path):
+        tmp_dir_path = tmp_dir_path / "action_cls"
+        otx_export_testing(template, tmp_dir_path, half_precision=True)
+
+    @e2e_pytest_component
+    @pytest.mark.skipif(TT_STABILITY_TESTS, reason="This is TT_STABILITY_TESTS")
+    @pytest.mark.parametrize("template", templates, ids=templates_ids)
+    def test_otx_export_onnx(self, template, tmp_dir_path):
+        tmp_dir_path = tmp_dir_path / "action_cls"
+        otx_export_testing(template, tmp_dir_path, half_precision=False, is_onnx=True)
```

Comment:
Don't we support the ONNX export + Half-Precision?

Reply:
No, we don't support this. That question required additional investigation for each algorithm, because now such export fails or generates an fp32 model. Also, this feature is not required from the Geti side.

Reply:
Got it, thanks